x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/expression/function/scalar/conditional/CaseTests.java

```diff
@@ -34,24 +35,31 @@
 import static org.hamcrest.Matchers.startsWith;
 
 public class CaseTests extends AbstractScalarFunctionTestCase {
-    private static final List<DataType> TYPES = List.of(
-        DataType.KEYWORD,
-        DataType.TEXT,
-        DataType.BOOLEAN,
-        DataType.DATETIME,
-        DataType.DATE_NANOS,
```

Comment:
This one caught me by surprise...

Reply:
Oh neat! if we're not on snapshot we can't use DATE_NANOS.....

Reply:
Huh.  I wonder why we didn't catch a problem here when I added the data type and feature flag.  I did put the `test-release` tag on https://github.com/elastic/elasticsearch/pull/110205